extensions/ql-vscode/src/remote-queries/remote-queries-markdown-generation.ts

```diff
@@ -25,8 +34,37 @@ export function generateMarkdown(query: RemoteQuery, analysesResults: AnalysisRe
     }
     files.push(lines);
   }
+  files.push(summaryLines);
   return files;
+}
+
+export function generateMarkdownSummary(query: RemoteQuery): MarkdownFile {
+  const lines: MarkdownFile = [];
+  // Title
+  lines.push(`## Results for "${query.queryName}"`);
```

Comment:
Using `##` looks a bit too big when I look at the Markdown in my VS Code, and the sample gist seems to be using `###`. Shall we add an extra `#`?